asset_bundles/wine_quality_model.yaml

```
@@ -0,0 +1,98 @@
+      tasks:
+        - task_key: "preprocessing"
+          job_cluster_key: "wine-quality-cluster"
+          spark_python_task:
+            python_file: "../week5/preprocess.py"
+            parameters:
+              - "--root_path"
+              - ${var.root_path}
+              - "--env"
+              - ${var.env}
+          libraries:
+          - whl: ../dist/*.whl
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)